website/source/docs/agent/options.html.md

```diff
@@ -684,7 +679,8 @@ default will automatically work with some tooling.
 *   <a name="acl_replication_token_legacy"></a><a href="#acl_replication_token_legacy">`acl_replication_token`</a> -
     **Deprecated in Consul 1.4.0. See the [`acl.tokens.replication`](#acl_tokens_replication) field instead.**
     Only used for servers outside the [`primary_datacenter`](#primary_datacenter) running Consul 0.7 or later.
-    When provided, this will enable [ACL replication](/docs/guides/acl.html#replication) using this
+    When provided, this will enable [
```

Comment:
```suggestion
    When provided, this will enable [ACL replication](https://learn.hashicorp.com/consul/day-2-operations/acl-replication) using this
```